.github/workflows/pull.yml

```diff
@@ -0,0 +1,29 @@
+name: pull
+
+on:
+  push:
+    branches: [master, "release/*"]
+  pull_request:
+    branches: [master, "release/*"]
+
+concurrency:
+  group: ${{ github.workflow }}-${{ github.ref }}-${{ github.head_ref }}
+  cancel-in-progress: ${{ ! (github.ref == 'refs/heads/main' || startsWith(github.ref, 'refs/heads/release/')) }}
+
+jobs:
+  check-code:
+    uses: Lightning-AI/devtools/.github/workflows/check-code.yml@main
+
+  check-schema:
+    uses: Lightning-AI/devtools/.github/workflows/check-schema.yml@main
+    with:
+      azure-dir: ""
```

Comment:
let's move this config to its own folder
https://github.com/Lightning-AI/lightning-bolts/blob/master/azure-pipelines.yml

Reply:
Done!